config.go

```diff
@@ -25,6 +29,15 @@ func LoadConfigFromPath(path string) (Config, error) {
 	if dsn, found := os.LookupEnv("TIGERBLOOD_DSN"); found {
 		config.DatabaseDsn = dsn
 	}
+	if dbMaxOpenConns, found := os.LookupEnv("TIGERBLOOD_DB_MAX_OPEN_CONNS"); found {
+		dbMaxOpenConns, err := strconv.Atoi(dbMaxOpenConns)
+		if err != nil {
+			config.DatabaseMaxOpenConns = dbMaxOpenConns
```

Comment:
Currently I'm basically just showing a warning on main.go whenever the config fails to load, because on config.go, I just return the error the yaml parser returns. The parser will return an error even if it's just a single optional field missing. This is not very nice.

There isn't really a nice, declarative way to give configuration items a default value or make them optional in plain Go, and I'm not really a fan of having more code for configuration than code for actual service logic, which is where making our own config with default values, etc, will take us. Maybe it would be worth it to use something like https://github.com/spf13/viper for configuration, instead of hand rolling something similar?